include/libopencm3/stm32/g0/rcc.h

```diff
@@ -0,0 +1,948 @@
+	SCC_LPUART1 = _REG_BIT(RCC_APBSMENR1_OFFSET, 20),
+	SCC_USART4 = _REG_BIT(RCC_APBSMENR1_OFFSET, 19),
+	SCC_USART3 = _REG_BIT(RCC_APBSMENR1_OFFSET, 18),
+	SCC_USART2 = _REG_BIT(RCC_APBSMENR1_OFFSET, 17),
+	SCC_SPI2 = _REG_BIT(RCC_APBSMENR1_OFFSET, 14),
+	SCC_TIM7 = _REG_BIT(RCC_APBSMENR1_OFFSET, 5),
+	SCC_TIM6 = _REG_BIT(RCC_APBSMENR1_OFFSET, 4),
+	SCC_TIM3 = _REG_BIT(RCC_APBSMENR1_OFFSET, 1),
+	SCC_TIM2 = _REG_BIT(RCC_APBSMENR1_OFFSET, 0),
+
+	SCC_ADC = _REG_BIT(RCC_APBSMENR2_OFFSET, 20),
+	SCC_TIM17 = _REG_BIT(RCC_APBSMENR2_OFFSET, 18),
+	SCC_TIM16 = _REG_BIT(RCC_APBSMENR2_OFFSET, 17),
+	SCC_TIM15 = _REG_BIT(RCC_APBSMENR2_OFFSET, 16),
+	SCC_TIM14 = _REG_BIT(RCC_APBSMENR2_OFFSET, 15),
+	SCC_USART1 = _REG_BIT(RCC_APBSMENR2_OFFSET, 14),
+	SCC_SPI1 = _REG_BIT(RCC_APBSMENR2_OFFSET, 12),
+	SCC_TIM1 = _REG_BIT(RCC_APBSMENR2_OFFSET, 11),
+	SCC_SYSCFG = _REG_BIT(RCC_APBSMENR2_OFFSET, 0),
+};
+
+enum rcc_periph_rst {
+	RST_GPIOF = _REG_BIT(RCC_IOPRSTR_OFFSET, 5),
+	RST_GPIOE = _REG_BIT(RCC_IOPRSTR_OFFSET, 4),
+	RST_GPIOD = _REG_BIT(RCC_IOPRSTR_OFFSET, 3),
+	RST_GPIOC = _REG_BIT(RCC_IOPRSTR_OFFSET, 2),
+	RST_GPIOB = _REG_BIT(RCC_IOPRSTR_OFFSET, 1),
+	RST_GPIOA = _REG_BIT(RCC_IOPRSTR_OFFSET, 0),
+
+	RST_RNG = _REG_BIT(RCC_AHBRSTR_OFFSET, 18),
+	RST_AES = _REG_BIT(RCC_AHBRSTR_OFFSET, 16),
+	RST_CRC = _REG_BIT(RCC_AHBRSTR_OFFSET, 12),
+	RST_FLASH = _REG_BIT(RCC_AHBRSTR_OFFSET, 8),
+	RST_DMA = _REG_BIT(RCC_AHBRSTR_OFFSET, 0),
+	RST_DMA1 = _REG_BIT(RCC_AHBRSTR_OFFSET, 0), /* Compatibility */
+
+	RST_LPTIM1 = _REG_BIT(RCC_APBRSTR1_OFFSET, 31),
+	RST_LPTIM2 = _REG_BIT(RCC_APBRSTR1_OFFSET, 30),
+	RST_DAC1 = _REG_BIT(RCC_APBRSTR1_OFFSET, 29),
+	RST_PWR = _REG_BIT(RCC_APBRSTR1_OFFSET, 28),
+	RST_DBG = _REG_BIT(RCC_APBRSTR1_OFFSET, 27),
+	RST_UCPD1 = _REG_BIT(RCC_APBRSTR1_OFFSET, 26),
+	RST_UCPD2 = _REG_BIT(RCC_APBRSTR1_OFFSET, 25),
+	RST_CEC = _REG_BIT(RCC_APBRSTR1_OFFSET, 24),
+	RST_I2C2 = _REG_BIT(RCC_APBRSTR1_OFFSET, 22),
+	RST_I2C1 = _REG_BIT(RCC_APBRSTR1_OFFSET, 21),
+	RST_LPUART1 = _REG_BIT(RCC_APBRSTR1_OFFSET, 20),
+	RST_USART4 = _REG_BIT(RCC_APBRSTR1_OFFSET, 19),
+	RST_USART3 = _REG_BIT(RCC_APBRSTR1_OFFSET, 18),
+	RST_USART2 = _REG_BIT(RCC_APBRSTR1_OFFSET, 17),
+	RST_SPI2 = _REG_BIT(RCC_APBRSTR1_OFFSET, 14),
+	RST_TIM7 = _REG_BIT(RCC_APBRSTR1_OFFSET, 5),
+	RST_TIM6 = _REG_BIT(RCC_APBRSTR1_OFFSET, 4),
+	RST_TIM3 = _REG_BIT(RCC_APBRSTR1_OFFSET, 1),
+	RST_TIM2 = _REG_BIT(RCC_APBRSTR1_OFFSET, 0),
+
+	RST_ADC = _REG_BIT(RCC_APBRSTR2_OFFSET, 20),
+	RST_TIM17 = _REG_BIT(RCC_APBRSTR2_OFFSET, 18),
+	RST_TIM16 = _REG_BIT(RCC_APBRSTR2_OFFSET, 17),
+	RST_TIM15 = _REG_BIT(RCC_APBRSTR2_OFFSET, 16),
+	RST_TIM14 = _REG_BIT(RCC_APBRSTR2_OFFSET, 15),
+	RST_USART1 = _REG_BIT(RCC_APBRSTR2_OFFSET, 14),
+	RST_SPI1 = _REG_BIT(RCC_APBRSTR2_OFFSET, 12),
+	RST_TIM1 = _REG_BIT(RCC_APBRSTR2_OFFSET, 11),
+	RST_SYSCFG = _REG_BIT(RCC_APBRSTR2_OFFSET, 0),
+};
+
+struct rcc_clock_scale {
+	enum rcc_osc sysclock_source;
+		
+	// PLL as sysclock source cfg
```

Comment:
wrong comment style

Reply:
indeed... and i've got them like that like all around ... sorry.